dora/integration/fuse/src/main/java/alluxio/fuse/AlluxioFuse.java

```diff
@@ -210,34 +226,70 @@ public static FuseUmountable launchFuse(FileSystemContext fsContext, FileSystem
       }
     }
 
-    final boolean debugEnabled = conf.getBoolean(PropertyKey.FUSE_DEBUG_ENABLED);
     if (conf.getBoolean(PropertyKey.FUSE_JNIFUSE_ENABLED)) {
       final AlluxioJniFuseFileSystem fuseFs
           = new AlluxioJniFuseFileSystem(fsContext, fs, fuseOptions);
+      return fuseFs;
+    } else {
```

Comment:
as a side question, should we remove jnr fuse in alluxio 300? @LuQQiu

Reply:
I thought @LuQQiu had a plan/task to remove the JNR implementation totally from Alluxio code base.

If this is not planned, maybe we can do it in this task.

Reply:
I think JNR can be removed, but maybe we can remove JNR in another PR.